scripts/gen_vimdoc.py

```diff
@@ -197,8 +203,8 @@
             'query.lua',
             'highlighter.lua',
             'languagetree.lua',
-            'health.lua',
         ],
+        'ignored_sections': ['health.lua'],
```

Comment:
instead of this, doxygen supports `EXCLUDE_PATTERNS`  https://github.com/neovim/neovim/blob/80a75fc81560269e5d36824ea1c1e328140f82c9/scripts/gen_vimdoc.py#L1142

maybe it works to add a `exclude_pattern` field similar to `file_patterns`